src/main/java/journeymap/client/api/util/PolygonHelper.java

```diff
@@ -63,4 +75,208 @@ public static MapPolygon createChunkPolygon(int chunkX, int y, int chunkZ)
 
         return new MapPolygon(sw, se, ne, nw);
     }
+
+    /**
+     * Given a collection of chunks, creates an Area that covers them.
+     *
+     * @param chunks The set of chunks.
+     * @return An Area of the corresponding block coordinates.
+     */
+    @Nonnull
+    public static Area createChunksArea(@Nonnull final Collection<ChunkPos> chunks)
+    {
+        final Area area = new Area();
+        for (final ChunkPos chunkPos : chunks)
+        {
+            area.add(new Area(new Rectangle(chunkPos.getMinBlockX(), chunkPos.getMinBlockZ(), 16, 16)));
+        }
+        return area;
```

Comment:
Should this just return the PolygonWithHoles? Or add another method that takes a collection of chunkpos and returns the PolygonWithHoles.

Reply:
The idea is that you could add more things to the `Area` than just chunks (or make a custom `Area` without calling anything in here, e.g. for a non-chunk-aligned `BlockPos` rect, or a non-rectangular polygon), and then you finish up with `createPolygonFromArea` regardless of how it was created.

But I could possibly add a convenience method that combines both steps.

Reply:
Thanks!